.github/workflows/contrib-tests.yml

```diff
@@ -5,6 +5,8 @@ on:
     tags:
       - v[0-9]+.[0-9]+.[0-9]+.*
   pull_request:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
```

Comment:
Should `ready_for_review` be included here? I'm not sure if there are other `types` that should be included maybe `edited`?

Reply:
If we add `ready_for_review` we should update the changelog workflow to match.

Reply:
Done for `ready_for_review`